src/languages/c-like.js

```diff
@@ -200,6 +201,58 @@ export default function(hljs) {
     ]
   };
 
+  var TEMPLATE_USE = {
+    begin: /</, end: />/,
+    keywords: CPP_KEYWORDS,
+    contains: EXPRESSION_CONTAINS.concat([
+      // Match left-shift to prevent it from creating a nested `TEMPLATE_USE`.
```

Comment:
I’m repeating this quite a lot. I’m wondering whether this could be simplified, or whether the `/<</` mode should be moved inside `EXPRESSION_CONTAINS`.

Reply:
It doesn't look that terrible to me...

> whether the /<</ mode should be moved inside EXPRESSION_CONTAINS.

It doesn't technically belong there since it's only relevant in the context of a template, so I think the more explicit way it is now is slightly better than throwing it in there with a comment.